_artifacts/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "asciidoc.antora.enableAntoraSupport": false
+}
```

Comment:
is this intentionally addded?  @emmurphy1

Reply:
Nope. It just appeared. I deleted it.